templates/eap-cd-sso-s2i.json

```diff
@@ -489,6 +504,10 @@
                             "name": "eap-cd-openshift:12"
                         },
                         "env": [
+                            {
+                                "name": "CUSTOM_INSTALL_DIRECTORIES",
+                                "value": "rh-sso/client_adapters/*"
```

Comment:
I think we should leave this as `extensions/*` and the mount point for the sso adapters should be changed to `"destinationDir": "./${CONTEXT_DIR}/extensions/sso-adapters`.  The trick here is that the glob will scan all directory children of extensions, which would allow other extensions to be added without modifying `CUSTOM_INSTALL_DIRECTORIES`, e.g. adding a db driver image mounted at `./${CONTEXT_DIR}/extensions/some-db`.  We could even consider specifying `extensions/*` as the default setting, which would allow users to just add image sources without having to do other modifications (e.g. adding a db driver after using the _basic_ templates).